src/SourceBuild/tarball/content/README.md

```diff
@@ -0,0 +1,160 @@
+This means it won't be short-lived anymore and we won't be reserving the right to delete and re-create it anymore.
+Other limitations might apply until the .NET 9 timeframe.
+See the [Unified Build roadmap](https://github.com/dotnet/arcade/blob/main/Documentation/UnifiedBuild) for more details.
+
+### Supported platforms / builds
+
+.NET Source-Build is supported on the oldest available .NET SDK feature update for each major release, and on Linux only.
+For example, if .NET `6.0.1xx`, `6.0.2xx`, and `7.0.1xx` feature updates are available from [dotnet.microsoft.com](https://dotnet.microsoft.com/en-us/download/dotnet/6.0), Source-Build will support `6.0.1xx` and `7.0.1xx`.
+
+For the latest information about Source-Build support for new .NET versions, please check our [GitHub Discussions page](https://github.com/dotnet/source-build/discussions) for announcements.
+
+It is expected that other platforms will be supported in the future.
+
+### Online build only
+
+Building the product offline is not fully working at the moment. The `--online` switch is needed when [building](#building) as not all dependencies are currently built from source.
+
+### Code flow
+For the time being, the source code only flows one way - from the individual repos into the VMR.
+Changes done to the VMR are not automatically mirrored back. More details on this process:
+
+- [Source Synchronization Process](https://github.com/dotnet/arcade/blob/main/Documentation/UnifiedBuild/VMR-Design-And-Operation.md#source-synchronization-process)
+- [Moving Code and Dependencies between the VMR and Development Repos](https://github.com/dotnet/arcade/blob/main/Documentation/UnifiedBuild/VMR-Design-And-Operation.md#moving-code-and-dependencies-between-the-vmr-and-development-repos)
+
+We expect the code flow to start working both ways in the .NET 9 timeframe.
+See the [Unified Build roadmap](https://github.com/dotnet/arcade/blob/main/Documentation/UnifiedBuild) for more details.
+
+### Contribution
+
+At this time, the VMR will not accept any changes outside of those necessary for enabling the supported builds to work.
+Please, make the changes in the respective development repositories (e.g., [dotnet/runtime](https://github.com/dotnet/runtime) or [dotnet/sdk](https://github.com/dotnet/sdk)) and they will get synchronized into the VMR automatically.
+
+## Goals
+
+The main purpose of this repository is to have all source code necessary to build the whole .NET product from source contained in any given commit.
+
+Further, the VMR eventually aims to become the place from which we release and service future versions of .NET.
+This is to reduce the complexity of the product construction process and thus enable partners and 3rd parties to easily build, test and modify .NET using their custom infrastructure.
+
+Furthermore, we hope to solve other problems that the current multi-repo setup brings:
+- Enable developers to make and test changes spanning multiple repositories.
```

Comment:
Are these in priority order?  They don't align with what I think the priority should be.  I see this one as being towards the bottom.

Reply:
They are not numbered but I guess the order matters the same. I re-shuffled them.